kerastuner/tuners/bayesian.py

```diff
@@ -107,21 +112,19 @@ def _populate_space(self, trial_id):
             return {'status': trial_lib.TrialStatus.RUNNING,
                     'values': values}
 
-        def _upper_confidence_bound(x):
-            x = x.reshape(1, -1)
-            mu, sigma = self.gpr.predict(x, return_std=True)
-            return mu - self.beta * sigma
-
         optimal_val = float('inf')
         optimal_x = None
         num_restarts = 50
+        y_max = self.get_best_trials()[0].score
```

Comment:
The ```score``` is sometimes minimized and sometimes maximized. Also we do score normalization before fitting in the GPR

Instead this should use the vectorized trials, like:

```python
y_max = np.max(y)
```